samples/main.cpp

```diff
@@ -170,6 +170,10 @@ int main(int argc, char* argv[])
     }
 
     novelChanRect->getTransform().setRotation(rotation);
+
+    if (runner.getInteractionService().lock()->getKeyState(NovelRT::Input::KeyCode::W) != NovelRT::Input::KeyState::Idle) {
+      console.logInfoLine("W is not idle!");
```

Comment:
Seems like this will be a lot of logging. We might want to revisit this when we clean up the sample in a couple weeks.